SpectralLineCrawler.cc

```diff
@@ -0,0 +1,142 @@
+#include "SpectralLineCrawler.h"
```

Comment:
Could this class not go in the Table subdirectory? We should avoid putting anything in the top-level directory unless it's very general, and this looks like it's specific to catalogues.